src/main/java/com/redhat/rhjmc/containerjfr/commands/internal/UploadRecordingCommand.java

```diff
@@ -110,7 +110,7 @@ public String getName() {
     public void execute(String[] args) throws Exception {
         String targetId = args[0];
         String recordingName = args[1];
-        String datasourceUrl = env.getEnv(GRAFANA_DATASOURCE_ENV);
+        String datasourceUrl = env.getEnv(GRAFANA_DATASOURCE_ENV).concat("/load");
```

Comment:
Does this work if the user sets the env var to ex. `http://localhost:8080/`, with a trailing slash?

Reply:
Yes, tested just now and it works.

Reply:
Cool. I thought a double-slash might break things, since it could be interpreted as a null or empty string as the first path segment. If the Vertx webclient and the jfr-datasource both accept this then no worries.

Reply:
Yeah, when I was working on the web client I had a trailing slash at first and I noticed that even though the logs said it was uploading to `http://localhost:8080//load` which seemed like it'd break for sure, it was working. I guess it's cool that that works; makes stuff like this a lot easier.

Reply:
It isn't too hard to clean it up anyway. There are cleaner ways using URIBuilder and such, but with just plain JDK classes:

`new URL(env.getEnv(GRAFANA_DATASOURCE_ENV)).toURI().resolve("/load").normalize().toString()`

should do it.